client/src/index.js

```diff
@@ -28,6 +28,7 @@ store.subscribe(
           sortingOption: addedCourses.spring.sortingOption
         }
       },
+      fetchedCourses: loadState().fetchedCourses,
```

Comment:
There are no fetchedCourses in redux store when a user opens a new course evaluation tab, so fetchedCourses can only be loaded from LocalStorage instead.
Also I just made a commit so that if the user is not in evaluation page and fetchedCourses haven't saved to LocalStorage, courses can be saved from redux store to LocalStorage .
Problems for a user in the evaluation page but fetchedCourses haven't saved to LocalStorage (perhaps the user haven't use WasedaTime before and just received a link directly to the evaluation page) will be solved in another commit later.